### PR TITLE
Add baz endpoint benchmarks

### PR DESCRIPTION
**Pull request by ChuntaoLu**
FYI some local benchmark runs on baz endpoint:

```
➜  zanzibar git:(lu.bench) wrk -t12 -c400 -d30s -s ./benchmarks/baz_call_16B.lua http://localhost:8093/baz/call
Running 30s test @ http://localhost:8093/baz/call
  12 threads and 400 connections
  Thread Stats   Avg      Stdev     Max   +/- Stdev
    Latency    20.86ms   10.58ms 148.98ms   72.16%
    Req/Sec     1.59k   234.78     3.06k    69.06%
  571344 requests in 30.09s, 52.31MB read
  Socket errors: connect 0, read 234, write 0, timeout 0
Requests/sec:  18990.67
Transfer/sec:      1.74MB
➜  zanzibar git:(lu.bench) wrk -t12 -c400 -d30s -s ./benchmarks/baz_call_4k.lua http://localhost:8093/baz/call
Running 30s test @ http://localhost:8093/baz/call
  12 threads and 400 connections
  Thread Stats   Avg      Stdev     Max   +/- Stdev
    Latency    23.46ms   12.94ms 164.28ms   70.00%
    Req/Sec     1.40k   190.07     2.32k    71.50%
  503917 requests in 30.09s, 46.13MB read
  Socket errors: connect 0, read 236, write 0, timeout 0
Requests/sec:  16747.82
Transfer/sec:      1.53MB
```
Benchmarks on contacts endpoint for reference:
```
➜  zanzibar git:(lu.bench) wrk -t12 -c400 -d30s -s ./benchmarks/contacts_10B.lua http://localhost:8093/contacts/foo/contacts
Running 30s test @ http://localhost:8093/contacts/foo/contacts
  12 threads and 400 connections
  Thread Stats   Avg      Stdev     Max   +/- Stdev
    Latency    16.32ms    8.30ms 181.48ms   89.75%
    Req/Sec     2.05k   345.04     4.18k    72.54%
  736287 requests in 30.10s, 80.75MB read
  Socket errors: connect 0, read 238, write 0, timeout 0
Requests/sec:  24461.72
Transfer/sec:      2.68MB
➜  zanzibar git:(lu.bench) wrk -t12 -c400 -d30s -s ./benchmarks/contacts_4KB.lua http://localhost:8093/contacts/foo/contacts
Running 30s test @ http://localhost:8093/contacts/foo/contacts
  12 threads and 400 connections
  Thread Stats   Avg      Stdev     Max   +/- Stdev
    Latency    29.14ms   12.43ms 179.13ms   79.54%
    Req/Sec     1.13k   133.86     1.87k    70.42%
  404767 requests in 30.07s, 44.39MB read
  Socket errors: connect 0, read 242, write 0, timeout 0
Requests/sec:  13459.60
Transfer/sec:      1.48MB
```

### CODE REVIEW

File: benchmarks/benchserver/main.go [hunk 1]
```diff
@@ -40,23 +45,54 @@ func main() {
 			zap.InfoLevel,
 		),
 	)
-	backend := testBackend.CreateHTTPBackend(PORT)
-	err := backend.Bootstrap()
+	serverTChannel(logger)
+	serveHTTP(logger)
```
Comment (Raynos) [inline benchmarks/benchserver/main.go]: nitpick: move the Wait() call up here outside the `serve()` call.

File: codegen/template_bundle/template_files.go [hunk 2]
```diff
@@ -92,7 +92,7 @@ func endpointTmpl() (*asset, error) {
 		return nil, err
 	}
 
-	info := bindataFileInfo{name: "endpoint.tmpl", size: 6955, mode: os.FileMode(420), modTime: time.Unix(1492819038, 0)}
+	info := bindataFileInfo{name: "endpoint.tmpl", size: 6955, mode: os.FileMode(420), modTime: time.Unix(1492822079, 0)}
```
Comment (Raynos) [inline codegen/template_bundle/template_files.go]: Need to figure out why these timestamps change :(

File: benchmarks/benchserver/main.go (hunk 1 above)
Comment (ChuntaoLu) [inline benchmarks/benchserver/main.go]: Done

File: codegen/template_bundle/template_files.go (hunk 2 above)
Comment (ChuntaoLu) [inline codegen/template_bundle/template_files.go]: yeah, this is wired, no templates were modified...

**Pull request merged**